src/sqlfluff/utils/reflow/sequence.py

```diff
@@ -74,6 +79,58 @@ def get_fixes(self) -> List[LintFix]:
         """Get the current fix buffer."""
         return self.embodied_fixes
 
+    def get_results(self) -> List[LintResult]:
+        """Generate a list of LintResult from the current fix buffer.
+
+        For some fixes, we do a bit of a shuffle with the anchor so
+        that we reference the most sensible position in the source file
+        to make sense to the user. For example if we've due to create
+        something *after* a segment, then we'll hunt forward to the
+        segment *after that*, so that the position that appears in the
+        CLI is the position between them.
+
+        NOTE: This generates one result per fix. Depending on your use
+        case, this may not always be appropriate. In those cases, call
+        `ReflowSequence.get_fixes()` and construct :obj:`LintResult`
+        objects directly.
+        """
+        results = []
+        segments = None
+        for fix in self.get_fixes():
+            # Anchoring on the segment AFTER makes is a lot more
```

Comment:
The word "makes" here confuses me. Is this correct?

Reply:
I've revised this comment to:
```python
            # In the case of *creations*, even if we are actually
            # creating *after* something, it makes sense to anchor
            # on the segment which follows the creation point and not
            # the segment which precedes it. That's because when
            # displaying the position of a linting result in the CLI
            # we display the position of the *start* of the anchor.
            # By anchoring on the segment *following* the insertion point
            # we display the position in a way which is much more
            # interpretable by the user.
```

Is that clearer?

Reply:
That's good!